pkg/agent/sysadvisor/plugin/qosaware/resource/cpu/region/region_base.go

```diff
@@ -124,11 +130,17 @@ func NewQoSRegionBase(name string, ownerPoolName string, regionType types.QoSReg
 		metaReader: metaReader,
 		metaServer: metaServer,
 		emitter:    emitter,
+
+		enableBorweinModel: conf.PolicyRama.EnableBorwein,
 	}
 
 	r.initHeadroomPolicy(conf, extraConf, metaReader, metaServer, emitter)
 	r.initProvisionPolicy(conf, extraConf, metaReader, metaServer, emitter)
 
+	if r.enableBorweinModel {
```

Comment:
will it be better to initial with a dummy `borweinController`? since it may cause many `if/else` in the future (and we already gots two now)